nginx.conf

```
@@ -2,6 +2,38 @@ upstream backend {
     server unix:/var/www/gunicorn.sock fail_timeout=0;
 }
 
+# Catch all, 404 in case a subdomain record exists, but we don't have the server block
+server {
+    server_name default_server;
+    listen 80;
+
+    root /var/www/html;
+
+    access_log /var/www/logs/csss-site-backend/nginx-access.log;
+    error_log /var/www/logs/csss-site-backend/nginx-error.log;
+
+    location / {
+        charset utf-8;
+        try_files 404.html;
+    }
+}
+
+# For pages under construction
+server {
+    server_name sfucsss.org;
+    listen 80;
+
+    root /var/www/html;
+
+    access_log /var/www/logs/csss-site-backend/nginx-access.log;
+    error_log /var/www/logs/csss-site-backend/nginx-error.log;
+
+    location / {
+        charset utf-8;
+        try_files construction.html;
```

Same here, i think this expects a 503 error code fallback?
```suggestion
        try_files /construction.html =503;
```